patroni/postgresql.py

```diff
@@ -1365,7 +1371,10 @@ def follow(self, member, timeout=None):
         if primary_conninfo:
             recovery_params['primary_conninfo'] = primary_conninfo
         if self.use_slots:
-            recovery_params['primary_slot_name'] = slot_name_from_member_name(self.name)
+            required_name = member is not None and member.data.get('replication_slot')
+            name = required_name or slot_name_from_member_name(self.name)
```

Comment:
Unfortunately it doesn't fix a problem.
Lets assume that we want to follow the remote cluster without using replication slots.
So we have no `replication_slot defined` in the `standby_cluster`, but according to this line it will set `primary_slot_name` to the `slot_name_from_member_name(self.name)`.

Actually the same function should be applied to the `required_name`

Reply:
Hm...looks like it means that this function should know whether this particular member is a part of standby cluster or not, am I right?